### PR TITLE
Added and an input for User Advice on each iteration of the loop

### PR DESCRIPTION
**Pull request by fjrdomingues**
Since I often don't agree with the direction that autogpt is going, or because I can save some time (and tokens) by giving him advice, I added an input for each iteration where the user can add advice. Autogpt will then take that advice into account for the next iteration.

I took the opportunity to also test my "autogpt" for code: https://github.com/fjrdomingues/autopilot

### CODE REVIEW

**Comment by Josedapker** [comment]
Very helpful. Thank you

File: .env.template
Comment (nponeccop) [inline .env.template:1]: This change is unrelated. Remove from this PR. Keep your PRs atomic to improve the chances of merging.

File: .gitignore
Comment (nponeccop) [inline .gitignore:1]: This change is unrelated. Remove from this PR. Keep your PRs atomic to improve the chances of merging.

File: autopilot
Comment (nponeccop) [inline autopilot:1]: Using submodules is a big design decision, making it harder for people to clone. This should be discussed apart.

File: scripts/main.py [hunk 1]
```diff
@@ -354,11 +358,21 @@ def parse_arguments():
...
+            print_to_console("SYSTEM: ", Fore.YELLOW, newResult)
+        else:
+            full_message_history.append(chat.create_chat_message("system", result))
+            print_to_console("SYSTEM: ", Fore.YELLOW, result)
     else:
         full_message_history.append(
             chat.create_chat_message(
                 "system", "Unable to execute command"))
         print_to_console("SYSTEM: ", Fore.YELLOW, "Unable to execute command")
+
```
Comment (nponeccop) [inline scripts/main.py]: A non-essential whitespace change. Submit it apart.

**Review by nponeccop — CHANGES_REQUESTED** [review]
Make it more focused. Remove the unrelated changes in gitignore and the .env.template.

**Comment by nponeccop** [comment]
@fjrdomingues Make it more focused and resolve the conflicts

**Review by nponeccop — APPROVED** [review]

**Comment by nponeccop** [comment]
@fjrdomingues Conflicts again. We are merging minor PRs

**Review by nponeccop — APPROVED** [review]

**Comment by nponeccop** [comment]
@fjrdomingues There are conflicts again (due to the merged PR batch)

**Comment by Torantulino** [comment]
Just to clarify, how is this different to the "Human Feedback" that was added in #133?:

image

**Comment by fjrdomingues** [comment]
> Just to clarify, how is this different to the "Human Feedback" that was added in #133?:
> 
> image

Hum. It seems that other PR already solved the same problem! Good catch.
I’ll close this PR.

**Pull request closed without merging**